### PR TITLE
Add support for downloading woff2 fonts

### PR DESCRIPTION
**Pull request by JakeChampion**
Depends on this being merged and released first https://github.com/Financial-Times/o-fonts-assets/pull/19

### CODE REVIEW

File: src/scss/_mixins.scss [hunk 1]
```diff
@@ -145,7 +145,7 @@
 		}
 
 		@font-face {
-			src: url($o-fonts-path + '#{$family}-#{$font-suffix}' + '.woff');
```
Comment (nickcolley) [inline src/scss/_mixins.scss:148]: Not sure if it's important but should we consider including `format('woff2')` etc in these?

**Comment by notlee** [comment]
we'll need to update the `$o-fonts-path` too, which will cause (more) problems for page kits preloading as it will use a different path and double download the fonts 😬 

https://financialtimes.slack.com/archives/C3TJ6KXEU/p1590161366213400

File: README.md [hunk 2]
```diff
@@ -136,13 +136,13 @@ In this example we register a custom font "MyFont" with sans fallback `MyFont, s
     (weight: bold, style: normal)
 )) {
     @font-face {
-        src: url('MyFont-Thin.woff');
+        src: url('MyFont-Thin.woff2'), url('MyFont-Thin.woff');
```
Comment (notlee) [inline README.md:139]: does the example need updating with `format`?

File: README.md [hunk 3]
```diff
@@ -136,13 +136,13 @@ In this example we register a custom font "MyFont" with sans fallback `MyFont, s
     (weight: bold, style: normal)
 )) {
     @font-face {
-        src: url('MyFont-Thin.woff');
+        src: url('MyFont-Thin.woff2') format(woff2), url('MyFont-Thin.woff') format(woff);
```
Comment (JakeChampion) [inline README.md:139]:
```suggestion
        src: url('MyFont-Thin.woff2') format('woff2'), url('MyFont-Thin.woff') format('woff');
```

File: README.md [hunk 4]
```diff
@@ -136,13 +136,13 @@ In this example we register a custom font "MyFont" with sans fallback `MyFont, s
...
     @font-face {
-        src: url('MyFont-Thin.woff');
+        src: url('MyFont-Thin.woff2') format('woff2'), url('MyFont-Thin.woff') format('woff');
         font-family: MyFont;
         font-weight: 100;
         font-style: normal;
     }
     @font-face{
-        src: url('MyFont-Bold.woff');
+        src: url('MyFont-Bold.woff2') format(woff2), url('MyFont-Bold.woff') format(woff);
```
Comment (JakeChampion) [inline README.md:145]:
```suggestion
        src: url('MyFont-Bold.woff2') format('woff2'), url('MyFont-Bold.woff') format('woff');
```

File: README.md (hunk 2 above)
Comment (JakeChampion) [reply README.md]: Done

**Pull request merged**